content/posts/irods-development-update-february-2024.md

```diff
@@ -0,0 +1,279 @@
+Title: iRODS Development Update: February 2024
+Date: 2024-02-26 12:00
+Author: Kory Draughn
+Slug: irods-development-update-february-2024
+Status: published
+
+
+Hello reader,
```

Comment:
this feels new...

Reply:
Happy to change it.

Reply:
Is `Hello everyone,` more appropriate/preferred?

Reply:
it's perfectly fine - just made me smile.